hrui/hrui.php

```diff
@@ -934,3 +936,113 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+function _getMenuKeyMax($menuArray) {
+  $max = array(max(array_keys($menuArray)));
+  foreach($menuArray as $v) {
+    if (!empty($v['child'])) {
+      $max[] = _getMenuKeyMax($v['child']);
+    }
+  }
+  return max($max);
+}
+
+function civicrm_civicrm_navigationMenu(&$menu) {
+  // Create New Help Menu
+  _createHelpMenu($menu);
+
+  // Create New Developer Menu
+  _createDeveloperMenu($menu);
+}
+
+/**
+ * Creates Help Menu in navigation bar
+ *
+ * @param Array $menu
+ */
+function _createHelpMenu(&$menu) {
+  $maxKey = _getMenuKeyMax($menu);
+
+  $menu[$maxKey+1] = array(
+    'attributes' => array(
+      'label'      => 'Help',
+      'name'       => 'Help',
+      'url'        => null,
+      'permission' => 'access CiviCRM',
+      'navID'      => $maxKey + 1,
+      'active'     => 1
+    )
+  );
+
+  _hrui_civix_insert_navigation_menu($menu, 'Help', array(
+    'label' => ts('User Guide'),
+    'name' => ts('User Guide'),
+    'url' => 'http://civihr-documentation.readthedocs.io/en/latest/',
+    'permission' => 'access CiviCRM'
+  ));
+
+  _hrui_civix_insert_navigation_menu($menu, 'Help', array(
+    'label' => ts('CiviHR website'),
+    'name' => ts('CiviHR website'),
+    'url' => 'https://www.civihr.org/',
+    'permission' => 'access CiviCRM'
+  ));
+
+  _hrui_civix_insert_navigation_menu($menu, 'Help', array(
+    'label' => ts('Get support'),
+    'name' => ts('Get support'),
+    'url' => 'https://www.civihr.org/support',
+    'permission' => 'access CiviCRM'
+  ));
+}
+
+/**
+ * Creates Developer Menu in navigation bar
+ *
+ * @param Array $menu
+ */
+function _createDeveloperMenu(&$menu) {
+  $maxKey = _getMenuKeyMax($menu);
```

Comment:
Same as `_createHelpMenu`, if it works without `_getMenuKeyMax` we can remove it.